src/function/table.rs

```diff
@@ -182,13 +182,18 @@ impl Table {
     }
 
     /// One more than the maximum (potentially) valid offset into the table.
-    pub(crate) fn len(&self) -> usize {
+    pub(crate) fn num_offsets(&self) -> usize {
         self.vals.len()
     }
 
+    /// One more than the actual valid offset (not including stale) into the table.
```

Comment:
Is this comment right?

Reply:
I think so? The table len not including stale should be the actual valid offset?